html/js/main.js

```diff
@@ -140,8 +140,16 @@
     });
   };
 
+  function changeSiteHeaderImage(){
+    //random 0-9 accoring to /img/backgrounds
+    const rand = Math.floor(Math.random() * 10)
```

Comment:
We can add more images by bumping this number. I think we can just grab the first 6 since 7-9 are placeholders!

Reply:
Also above, the image for background 9 is the wrong name. It should be `html/img/backgrounds/9.jpg` but somehow it has an extra `]`.

Reply:
However I won't save the placeholders in the repo. I would just use the unique ones for now, and we can tune these numbers.